### PR TITLE
Extension 1.0.5 version release

### PR DESCRIPTION
**Pull request by kensium**

### CODE REVIEW

**Comment by kensium** [comment]
**Release Notes:**

1. Magento 2.1.9 Support
2. Magento 2.2.0 Support
3. Magento 2.2.1 Support

_**Bug Fixes:**_
1. WebExtend Multiple Go Commands error if Ajax disabled
2. Mapping Screen showing up blank Page in case of exception
3. Customer Field Mapping Issue On Multi Store
4. WebExtend not shown Cart command if the cart is empty
5. Subscriber Bulk Export - 500 Error on cloud server
6. Fastly Caching Broken with cacheable = false
7. Transactional Emails on Multistore
8. Customer Mapping issue on Multistore
9. Incorrect refund amount & quantity in SI export
10. Error on checkout due to subscription script
11. Customer address is not synced For multi website
12. Newsletter subscription / Guest Order SetEmail is displayed when Set Customer Id is configured in WebExtend
13. Sku displayed at cart instead of Product id when Item Unique Identification in WebExtend
14. Cron Customer export WebDAV CSV file not identical with Initial DB Load
15. Date filters in buk export changed to Magento date & time
16. WebExtend Purchase command formating issue
17. FTP timeout issue when exporting product catalog & sales
18. Removed duplicate columns in the Initial DB load for email address

File: emarsys/app/code/Emarsys/Emarsys/Block/Adminhtml/Scheduler/Renderer/ViewButton.php [hunk 1]
```diff
@@ -42,6 +42,6 @@ public function render(DataObject $row)
     {
         $url = 'logs/grid/index/job_code/'.$row['job_code'].'/schedule_id/'.$row['id'].'/store/'.$row['store_id'];
         $url = $this->backendHelper->getUrl($url);
-         printf("<a href='".$url."'><div style='color:#EB5202 ;text-decoration: underline;text-decoration-color:#EB5202;'>View</div></a>");
+        printf("<a href='%s'><div style='color:#EB5202 ;text-decoration: underline;text-decoration-color:#EB5202;'>View</div></a>", $url);
```
Comment (romastepa) [inline emarsys/app/code/Emarsys/Emarsys/Block/Adminhtml/Scheduler/Renderer/ViewButton.php]: there is no translates

File: emarsys/app/code/Emarsys/Emarsys/Block/Adminhtml/Scheduler/Renderer/ViewButton.php [hunk 2]
```diff
@@ -42,6 +42,6 @@ public function render(DataObject $row)
     {
         $url = 'logs/grid/index/job_code/'.$row['job_code'].'/schedule_id/'.$row['id'].'/store/'.$row['store_id'];
         $url = $this->backendHelper->getUrl($url);
```
Comment (romastepa) [inline emarsys/app/code/Emarsys/Emarsys/Block/Adminhtml/Scheduler/Renderer/ViewButton.php]: why you set parameters in a such strange way?
$this->backendHelper->getUrl($route = '', $params = [])

File: emarsys/app/code/Emarsys/Emarsys/Block/Adminhtml/Scheduler/Renderer/ViewButton.php (hunk 1 above)
Comment (romastepa) [inline emarsys/app/code/Emarsys/Emarsys/Block/Adminhtml/Scheduler/Renderer/ViewButton.php]: there is no translates

**Pull request merged**